src/components/SwagLandingCard.vue

```diff
@@ -24,14 +29,27 @@
     margin: 0;
     line-height: 133%;
   }
+.flex {
+    display: flex;
+    flex-direction: col;
+    align-items: center;
+  }
+.rounded-l-lg {
```

Comment:
This should be written as `class="rounded-l-[0.375rem]"` or when defined in css class as `@apply rounded-l-[0.375rem]` - see [Arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties).